lib/engines/content_object_store/test/functional/sentry_tags_test.rb

```diff
@@ -1,28 +1,32 @@
 require "test_helper"
 require "sentry/test_helper"
 
-class ContentObjectStore::SentryTagsTest < ActionDispatch::IntegrationTest
+class ContentObjectStore::SentryTagsTest < ActionController::TestCase
+  tests ContentObjectStore::ContentBlock::DocumentsController
   include Sentry::TestHelper
 
   setup do
     setup_sentry_test
     feature_flags.switch!(:content_object_store, true)
     login_as_admin
+    @routes = ContentObjectStore::Engine.routes
```

Comment:
I couldn't see what was using `@routes` so wanted to why we need this but removing it definitely fails:

```
ContentObjectStore::SentryTagsTest#test_throwing_an_error_includes_tags:
ActionController::UrlGenerationError: No route matches {:action=>"index", :controller=>"content_object_store/content_block/documents"}
```

This feels too mysterious. Might be worth a comment to explain "We have no idea what uses this instance variable but removing this makes it flakey"

Reply:
Yeah, that's a thing related to Functional Tests - they don't work out of the box in engines. I'll add a comment